ui/app/components/clients/page/counts.ts

```diff
@@ -141,17 +141,17 @@ export default class ClientsCountsPageComponent extends Component<Args> {
   }
 
   get activityForNamespace() {
+    // TODO: always return activity for namespace. If no namespace selected, use the current
```

Comment:
Is this a TODO for this PR or for a follow-on?

Reply:
Follow-on, this will be addressed with the filter bar work